snap/info.go

```diff
@@ -643,28 +643,42 @@ func (s *Info) CommonDataSaveDir() string {
 	return CommonDataSaveDir(s.InstanceName())
 }
 
-// DataHomeGlob returns the globbing expression for the snap directories in use
-func DataHomeGlob(opts *dirs.SnapDirOptions) string {
+// DataHomeGlobs returns a slice of globbing expressions for the snap directories in use
+func DataHomeGlobs(opts *dirs.SnapDirOptions) []string {
```

Comment:
this could move to the dirs package,  also in the dirs package SnapDataHomeGlob and HiddenSnapDataHomeGlob should be removed as they are not used anymore